### ISSUE TITLE
Issue in Technical Notes about versioning of constraints

### ISSUE THREAD

**Issue by dosse** [sdmx-twg/sdmx-im#23]
In the [SDMX STANDARDS: SECTION 6 TECHNICAL NOTES Version 3.0](https://sdmx.org/wp-content/uploads/SDMX_3-0-0_SECTION_6_FINAL-1_0.pdf), page 28, line 897 it currently says:

> A Constraint may reference many Dataflows or Metadataflows, the addition of more references to flow objects does not version the Constraint. This is because the Constraints are not properties of the flows – they merely make references to them.

I believe that the statement "_**the addition of more references to flow objects does not version the Constraint**_" is incorrect. The given justification "_they merely make references to them_" would imply that any artefact that references other artefacts would not be versionable, which is fully in opposite with the purpose of semantic versioning.

Actually, whenever a constraint changes (even because it references (constrains) other artefacts) then it absolutely needs to increase its version because this would be the only mean to indicate to others that a change has happened and that an another system wishing to synchronise its artefact copies needs to get the new version!

The SDMX-TWG TF2 working document on "Reorganising Constraints" actually stated:

> Any changes that include the following, result into minor version increase:
> –	Adding new Constrainables. ...

**Comment by egreising** [comment]
I agree with "signalling" the modification with a version change, but should not it be a "patch", not "minor"?

**Comment by stratosn** [comment]
Agreed to implement it. It is a bug in the documentation.
